kolibri/plugins/user/assets/src/actions.js

```diff
@@ -1,36 +1,34 @@
-// const coreApp = require('kolibri');
-// const logging = require('kolibri.lib.logging');
-
-// const FacilityUserResource = coreApp.resources.FacilityUserResource;
-// const TaskResource = coreApp.resources.TaskResource;
-
-// const coreActions = require('kolibri.coreVue.vuex.actions');
-// const ConditionalPromise = require('kolibri.lib.conditionalPromise');
 const constants = require('./state/constants');
 const PageNames = constants.PageNames;
-// const samePageCheckGenerator = require('kolibri.coreVue.vuex.actions').samePageCheckGenerator;
 const SignUpResource = require('kolibri').resources.SignUpResource;
 const coreActions = require('kolibri.coreVue.vuex.actions');
 const coreGetters = require('kolibri.coreVue.vuex.getters');
-
-// ================================
-// USER ACTIONS
+const router = require('kolibri.coreVue.router');
 
 
 function redirectToHome() {
   window.location = '/';
 }
 
-/**
- * Actions
- *
- * These methods are used to update client-side state
- */
+function showRoot(store) {
```

Comment:
Hmm interesting. So root would be taking the place of both `SignIn` and `Profile`?